packages/latex/src/main.rs

```diff
@@ -0,0 +1,222 @@
+use serde_json::{from_str, json, Value};
+use std::{
+    env,
+    fmt::Write,
+    io::{self, Read},
+};
+
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+
+    let Some(action) = args.get(0) else {
+        eprintln!("No action was provided.");
+        return;
+    };
+
+    match action.as_str() {
+        "manifest" => print!("{}", &manifest()),
+        "transform" => {
+            let from = args.get(1).unwrap();
+            let format = args.get(2).unwrap();
+
+            if "latex" != format {
+                eprintln!("Output format not supported");
+                return;
+            }
+
+            print!("{}", transform(from));
+        }
+        other => eprintln!("Invalid action '{other}'"),
+    }
+}
+
+fn transform(from: &str) -> String {
+    let input: Value = {
+        let mut buffer = String::new();
+        io::stdin().read_to_string(&mut buffer).unwrap();
+        from_str(&buffer).unwrap()
+    };
+
+    match from {
+        "__bold" => transform_tag(input, "textbf"),
+        "__italic" => transform_tag(input, "emph"),
+        "__superscript" => transform_tag(input, "textsuperscript"),
+        "__subscript" => transform_tag(input, "textsubscript"),
+        "__underlined" => transform_tag(input, "underline"),
+        "__strikethrough" => transform_tag(input, "sout"), //fixme: needs a package to use
+        "__verbatim" => transform_block(input, "verbatim"),
```

Comment:
I think this should use the inline `\verb|...|` since this is the verbatim tag, which (most likely) doesn't span multiple lines and cannot span multiple paragraphs